src/hooks/usePastTransaction/usePastTransaction.tsx

```diff
@@ -33,7 +33,11 @@ export const usePastTransaction = (
         );
         setPastTransactionsResult(pastTransactionsResponse?.pastTransactions);
       } catch (error) {
-        Sentry.captureException(`Unable to fetch past transactions: ${ids}`);
+        Sentry.captureException(
+          `Unable to fetch past transactions: ${ids.map(id =>
+            id.replace(/\w/g, "*")
```

Comment:
Do we want to leave the last 4 chars for debugging?

Reply:
for nrics it doesn't make a difference i think, but yeah for the other types of identification

Reply:
What do you mean it doesn't make a difference ah

Reply:
Btw in the backend we use `string.padStart()` to do masking, maybe we want to be consistent with that~ Can check out https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Reply:
nice, thanks! was wondering what's the best way lol. 
i think the nrics would be masked in our backend and db anyway, so it's a bit harder to debug